core_v2/sources/domains.move

```diff
@@ -52,33 +57,27 @@ module aptos_names::domains {
     /// Reverse lookup registry not initialized. `init_reverse_lookup_registry_v1` must be called first
     const EREVERSE_LOOKUP_NOT_INITIALIZED: u64 = 18;
 
-    struct NameRecordKeyV1 has copy, drop, store {
-        subdomain_name: Option<String>,
-        domain_name: String,
+    /// Tokens require a signer to create, so this is the signer for the collection
+    struct CollectionCapabilityV2 has key, drop {
+        capability: SignerCapability,
     }
 
-    struct NameRecordV1 has copy, drop, store {
-        // This is the property version of the NFT that this name record represents.
-        // This is required to tell expired vs current NFTs apart.
-        property_version: u64,
-        // The time, in seconds, when the name is considered expired
+    #[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+    struct NameRecordV2 has key {
+        domain_name: String,
```

Comment:
How do you feel about we only have 1 `name`? if it's a domain then name is `jianyi`, if it's a subdoamin name is `a.jianyi`, if we have subsubdomain in the future, it's `a.b.jianyi`?

Reply:
if it's a subdomain and we want to get domain, we can parse it, this way we only have 1 field and it's always required

Reply:
if we remove both where do we store the name?

Reply:
oh right the `create_named_token` thing